packages/data/src/world/mm/overworld.yml

```diff
@@ -2517,12 +2517,12 @@
 "Zora Cape Near Hall":
   region: ZORA_CAPE
   exits:
-    "Zora Cape": "underwater_walking"
-    "Zora Hall Entrance": "underwater_walking"
+    "Zora Cape": "underwater_walking || can_dive_small"
```

Comment:
I feel like this should also require the human trick if the way here uses it. Of course, none of it is relevant until ER shuffles the entrance.